helpers_key.go

```diff
@@ -20,8 +22,8 @@ import (
 var defaultKeyFlags = []cli.Flag{
 	&cli.StringFlag{
 		Name:        "sec",
-		Usage:       "secret key to sign the event, as nsec, ncryptsec or hex, or a bunker URL",
-		DefaultText: "the key '1'",
+		Usage:       "secret key to sign the event, as nsec, ncryptsec or hex, or a bunker URL, it is better to use the environment variable NOSTR_SECRET_KEY ",
```

Comment:
I would prefer to keep a stable key rather than generating a random key each time, `1` is a reasonable choice